Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -160,23 +194,30 @@ class VDPortGroup : BaseDSC {
     #>
     [PSObject] GetDistributedSwitch() {
         <#
-        The Verbose logic here is needed to suppress the Exporting and Importing of the
-        cmdlets from the VMware.VimAutomation.Vds Module.
+            The Verbose logic here is needed to suppress the Verbose output of the Import-Module cmdlet
+            when importing the 'VMware.VimAutomation.Vds' Module.
         #>
         $savedVerbosePreference = $global:VerbosePreference
         $global:VerbosePreference = 'SilentlyContinue'
 
         try {
+            $getVDSwitchParams = @{
+                Server = $this.Connection
+                Name = $this.VdsName
+                Verbose = $false
+            }
             if ($this.Ensure -eq [Ensure]::Absent) {
-                return Get-VDSwitch -Server $this.Connection -Name $this.VdsName -ErrorAction SilentlyContinue
+                $getVDSwitchParams.ErrorAction = 'SilentlyContinue'
+                return Get-VDSwitch @getVDSwitchParams
             }
             else {
                 try {
-                    $distributedSwitch = Get-VDSwitch -Server $this.Connection -Name $this.VdsName -ErrorAction Stop
-                    return $distributedSwitch
+                    Write-VerboseLog -Message $this.RetrieveVDSwitchMessage -Arguments @($this.VdsName)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.